pages/Portable-Geo-Scanner

```diff
@@ -0,0 +1,11 @@
+Potrable Geo-Scanner is a tool added in the [GPS](https://github.com/Slimefun/Slimefun4/wiki/GPS) category. It is a portable version of a [GPS Geo-Scanner](https://github.com/Slimefun/Slimefun4/wiki/GPS-Geo-Scanner)
```

Comment:
Typo in portable, also missing `The` 

At the end of the line, also add the, `It is a portable version of the [GPS Geo Scanner]()`